pkg/steps/pod.go

```diff
@@ -125,9 +125,6 @@ func (s *podStep) SubTests() []*junit.TestCase {
 }
 
 func (s *podStep) Requires() (ret []api.StepLink) {
-	if s.clusterClaim != nil {
-		ret = append(ret, api.ClusterClaimLink(s.config.As))
```

Comment:
How do we ensure that pod step won't run before claim step completes (without ClusterClaimLink)?

Reply:
I see. The original step is replaced/wrapped with/by the claim step in the graph.

Reply:
If that is the case, we can remove `ClusterClaimLink`.

https://github.com/openshift/ci-tools/blob/67cf7191964a2c432b1dc5baa0873487b55bce2b/pkg/api/graph.go#L567